docker/windows.Dockerfile

```diff
@@ -8,7 +11,8 @@ RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -ldflags "${LDFL
 
```

Comment:
If the Dockerfile container RUN, then I think we cannot use buildx to build windows container image on a linux machine. The workaround is to remote go build outside as a separate step.

Reply:
Indeed, but the RUN command is being run on a Linux image (``golang:1.13.10-alpine3.10``), which is why the actual build itself succeeded. I have posted the logs from my build.